mailbox/api/src/test/java/org/apache/james/mailbox/model/Rfc4314RightsTest.java

```diff
@@ -108,7 +107,7 @@ public void exceptShouldReturnOriginWhenExceptingEmpty() throws UnsupportedRight
     
     @Test
     public void fullRightsShouldContainsAllRights() {
-        assertThat(full).containsOnly(
+        assertThat(full.list()).containsOnly(
```

Comment:
Why do you need this `.list()` ?

Reply:
Rights is not a collection. So I need to trick assertj. Otherwize we can do *isEqualTo*